moveit_experimental/moveit_jog_arm/src/jog_calcs.cpp

```diff
@@ -346,14 +346,17 @@ bool JogCalcs::cartesianJogCalcs(geometry_msgs::TwistStamped& cmd, JogArmShared&
   if (!applyVelocityScaling(shared_variables, mutex, delta_theta_,
                             velocityScalingFactorForSingularity(delta_x, svd, jacobian, pseudo_inverse)))
   {
-    has_warning_ = true;
     suddenHalt(outgoing_command_);
   }
 
+  publishStatus(status_);
```

Comment:
Why pass a local variable?

Reply:
I like to keep it explicit. OK, I'll change it to `publishStatus()`